packages/components/src/color-picker/index.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Internal dependencies
+ */
+import { LegacyAdapter as ColorPicker } from './legacy-adapter';
+
+export default ColorPicker;
```

Comment:
Let's keep the named export from the new component

```suggestion
export { LegacyAdapter as ColorPicker } from './legacy-adapter';
```

We can then modify `packages/components/src/index.js` to:

```js
export { ColorPicker } from './color-picker';
```